Makefile

```diff
@@ -39,4 +39,4 @@ clean:
 
 .PHONY: server
 server:
-	go run . serve
+	LW_DEBUG=1 go run . serve
```

Comment:
careful committing this, and all the other debug stuff

Reply:
makefile is only for development, where debug mode by default makes sense